src/main/java/org/jvnet/hudson/plugins/monitoring/HudsonMonitoringFilter.java

```diff
@@ -85,8 +85,15 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 				return;
 			}
 			if (!PLUGIN_AUTHENTICATION_DISABLED) {
-				// only the Hudson/Jenkins administrator can view the monitoring report
-				Jenkins.getInstance().checkPermission(Jenkins.ADMINISTER);
+				final boolean hasSystemReadPermission = SystemReadPermission.SYSTEM_READ != null
+						&& Jenkins.getInstance().hasPermission(SystemReadPermission.SYSTEM_READ);
```

Comment:
this should never be null, it will fall back to ADMINISTER on old Jenkins instances
```suggestion
				final boolean hasSystemReadPermission = Jenkins.getInstance().hasPermission(SystemReadPermission.SYSTEM_READ);
```